code/modules/gear_presets/survivors/survivors.dm

```diff
@@ -313,13 +313,13 @@ Standart Survivors :	/datum/equipment_preset/survivor/scientist,
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/hazardvest/yellow(new_human), WEAR_JACKET)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/head/hardhat/white(new_human), WEAR_HEAD)
 	new_human.equip_to_slot_or_del(new /obj/item/storage/backpack/satchel/eng(new_human), WEAR_BACK)
-	new_human.equip_to_slot_or_del(new /obj/item/tool/weldingtool/largetank(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/tool/weldingtool/largetank(new_human), WEAR_R_STORE)
 	new_human.equip_to_slot_or_del(new /obj/item/stack/sheet/plasteel/med_small_stack(new_human), WEAR_IN_BACK)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/glasses/welding(new_human), WEAR_EYES)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/gloves/yellow(new_human), WEAR_HANDS)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(new_human), WEAR_FEET)
-	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(new_human), WEAR_WAIST)
 	add_survivor_weapon_civilian(new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/storage/belt/utility/full(new_human), WEAR_WAIST)
```

Comment:
looking back at it, this will never equip because proc above always adds something to waist. Either remove it or give a possibility to not spawn in a waist from proc above.

Reply:
`add_survivor_weapon_civilian()` has a 60 percent chance to not do anything at all. If it gives you a weapon, it also gives you an appropriate ammo belt, and the utility belt will just delete itself. The reference images in the main PR body show this behavior. (second image the proc rolled no weapon so the toolbelt was equiped. third image the proc did roll a weapon so there is no toolbelt.) This behavior is not intuitive unless you are familiar with the particular proc so I will add a comment to clarify.

Reply:
Didnt notice a prob(60) :patrice: